client/signup/controller.js

```diff
@@ -100,6 +100,7 @@ export default {
 	redirectTests( context, next ) {
 		const currentFlowName = getFlowName( context.params );
 		currentFlowName === 'onboarding' && loadExperimentAssignment( 'refined_reskin_v1' );
+		currentFlowName === 'launch-site' && loadExperimentAssignment( 'hide_ecommerce_launch_site' );
```

Comment:
Hmm, could be placed a bit later than on load, like at the signup step? 
Ideally it is placed at "the UI just before the UI you are testing".

For example the `refined_reskin_v1` is here as it is just before the signup screen.

Reply:
It looks like the eligibility here (`currentFlowName === 'launch-site'`) doesn't match the eligibility in `ProvideExperimentData` (`isLaunchPage && isInSignup`), ideally they are the same.

Reply:
Thanks for the suggestion.

There is another flow 'new-launch' defined that could have the same isLaunchPage = true. Let's make sure it doesn't happen, although, I don't think the gutenboarding launch flow uses any of this code.